src/msgpack/StreamWrapper.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.IO;
+
+namespace TarantoolDnx.MsgPack
+{
+    public class StreamWrapper : IMsgPackReader, IMsgPackWriter, IDisposable
```

Comment:
Don't mix both IMsgPackReader & Writer in same implementation
Add smth that points to MsgPack in name of class